### PR TITLE
English Check

### PR DESCRIPTION
**Pull request by EliChin**
Checked for some inaccuracies regarding English.

Signed-off-by: Elizaveta Chichindaeva <elizaveta@nspcc.ru>

### CODE REVIEW

**Comment by KirillovDenis** [comment]
Could you rename the commit using the following format: `[#153] ...`?

File: README.md [hunk 1]
```diff
@@ -207,7 +207,7 @@ For example:
 $ neofs-cli -r 192.168.130.72:8080 -k 6PYLKJhiSub5imt6WCVy6Quxtd9xu176omev1vWYovzkAQCTSQabAAQXii container create --policy "REP 3" --basic-acl public --await
 ```
 
-If you launched nodes via [neofs-dev-env](https://github.com/nspcc-dev/neofs-dev-env)
+If you have launched, nodes via [neofs-dev-env](https://github.com/nspcc-dev/neofs-dev-env)
```
Comment (alexvanin) [inline README.md:210]: Isn't this comma redundant?

File: downloader/download.go [hunk 2]
```diff
@@ -166,7 +166,7 @@ func (r request) receiveFile(clnt *pool.Pool, objectAddress *address.Address) {
 			return
 		}
 
-		// reset payload reader since part of the data has been read
+		// reset payload reader since a part of the data has been read
```
Comment (alexvanin) [inline downloader/download.go:169]: Maybe it's better to replace `since` with `because`?

**Review by alexvanin — APPROVED** [review]

**Review by KirillovDenis — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (EliChin) [reply README.md:210]: It is

**Pull request merged**